src/Tizen.NUI.Scene3D/src/public/Controls/Camera.cs

```diff
@@ -133,15 +133,15 @@ public ProjectionModeType ProjectionMode
         /// </summary>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public float FieldOfView
+        public Radian FieldOfView
         {
             get
             {
-                return (float)GetValue(FieldOfViewProperty);
+                return new Radian((float)GetValue(FieldOfViewProperty));
             }
             set
             {
-                SetValue(FieldOfViewProperty, value);
+                SetValue(FieldOfViewProperty, value.ConvertToFloat());
```

Comment:
There's a warning message, could you fix this?
-> In externally visible method 'void Camera.FieldOfView.set', validate parameter 'value' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.

You can just add null check for 'value'.

Reply:
Thank you I fixed it and pushed again.
Please check again.